datasetcard.md

```diff
@@ -0,0 +1,118 @@
+- {lang_1}  # Example: en
+license: {license}  # Example: apache-2.0 or any license from https://hf.co/docs/hub/repositories-licenses
+tags:
+- {tag_0}  # Example: audio
+- {tag_1}  # Example: bio
+- {tag_2}  # Example: natural language understanding
+- {tag_3}  # Example: birds classification
+annotations_creators:
+- {creator_0}  # Example: crowdsourced
+- {creator_1}  # Example: found
+- {creator_3}  # Example: expert-generated
+- {creator_4}  # Example: machine-generated
+language_creators:
+- {creator_0}  # Example: crowdsourced
+- {creator_1}  # Example: found
+- {creator_3}  # Example: expert-generated
+- {creator_4}  # Example: machine-generated
+language_details:
+- {bcp47_lang_0}  # Example: fr-FR
+- {bcp47_lang_1}  # Example: en-US
+license_details: {license_details}  # Example: content or link to a license not present in https://hf.co/docs/hub/repositories-licenses
+pretty_name: {pretty_name}  # Example: SQuAD
+size_categories:
+- {size_0}  # Example: n<1K
+- {size_0}  # Example: 100K<n<1M
+source_datasets:
+- {source_dataset_0}  # Example: wikipedia
+- {source_dataset_1}  # Example: laion/laion-2b
+task_categories:  # Full list at https://github.com/huggingface/hub-docs/blob/main/js/src/lib/interfaces/Types.ts
+- {task_0}  # Example: question-ansering
+- {task_1}  # Example: image-classification
+task_ids:
+- {subtask_0}  # Example: extractive-qa
+- {subtask_1}  # Example: multi-class-image-classification
+paperswithcode_id: {paperswithcode_id}  # Dataset id on PapersWithCode (from the URL). Example for SQuAD: squad
+configs:  # Optional for datasets with multiple configurations like glue.
+- {config_0}  # Example for glue: sst2
+- {config_1}  # Example for glue: cola
+
+# Optional. This part can be used to store the feature types and size of the dataset to be used in python. This can be automatically generated using the datasets-cli.
+dataset_info:
+  features:
+    - name: {feature_name_0}    # Example: id
+      dtype: {feature_dtyp_e0}  # Example: int32
+    - name: {feature_name_1}    # Example: text
+      dtype: {feature_dtype_1}  # Example: string
+    - name: {feature_name_2}    # Example: image
+      dtype: {feature_dtype_2}  # Example: image
+    # Example for SQuAD:
+    # - name: id
+    #   dtype: string
+    # - name: title
+    #   dtype: string
+    # - name: context
+    #   dtype: string
+    # - name: question
+    #   dtype: string
+    # - name: answers
+    #   sequence:
+    #     - name: text
+    #       dtype: string
+    #     - name: answer_start
+    #       dtype: int32
+  config_name: {config_name}  # Example for glue: sst2
+  splits:
+    - name: {split_name_0}                  # Example: train
+      num_bytes: {split_num_bytes_0}        # Example for SQuAD: 79317110
+      num_examples: {split_num_examples_0}  # Example for SQuAD: 87599
+  download_size: {dataset_download_size}   # Example for SQuAD: 35142551
+  dataset_size: {dataset_size}             # Example for SQuAD: 89789763
```

Comment:
mention that this will be auto-computed soon? so maybe no need to add it?

Reply:
oh actually there's a comment above that says this is generated automatically. so i think it's fine